File: CrossTemplate.VB/CrossTemplate.Tizen/CrossTemplate.Tizen.vb
Public Class SampleApplication
    Inherits Xamarin.Forms.Platform.Tizen.FormsApplication

    Protected Overrides Sub OnCreate()
        MyBase.OnCreate()
        LoadApplication(New App())
    End Sub

End Class

Module Program

    Sub Main(args As String())
        Dim app = New SampleApplication()
        Xamarin.Forms.Platform.Tizen.Forms.Init(app)
        app.Run(args)
    End Sub

End Module

File: CrossTemplate.FSharp/CrossTemplate.Tizen/CrossTemplate.Tizen.fs
namespace CrossTemplate.Tizen

type CrossTemplate() =
    inherit Xamarin.Forms.Platform.Tizen.FormsApplication ()

    override this.OnCreate() =
        base.OnCreate()
        this.LoadApplication(new CrossTemplate.App())

module Program =

    [<EntryPoint>]
    let main args =
        let app = new CrossTemplate()
        Xamarin.Forms.Platform.Tizen.Forms.Init(app)
        app.Run(args)
        0
